doc/source.rst

```diff
@@ -181,12 +181,46 @@ File patterns
 File path patterns are used for include and omit, and for combining path
 remapping.  They follow common shell syntax:
```

Comment:
This seems like a good place to reference the corresponding features:
```suggestion
File path patterns are used for :ref:`include <config_run_include>` and
:ref:`omit <config_run_omit>`, and for :ref:`combining path remapping
<cmd_combine_remapping>`.  They follow common shell syntax:
```

Reply:
_Shameless plug:_ I looked up the references through my little static site generator for intersphinx-enabled docs: https://webknjaz.github.io/intersphinx-untangled/coverage.rtfd.io/

Reply:
done.

Reply:
Did you force-push on top after that?

Reply:
I don't know what you mean.  The links are now on master.

Reply:
You're right, sorry about the noise! GitHub must've been showing me outdated diffs due to its infamous eventual consistency. And then, I've confused myself even more by clicking on the readme badge. Now I realize that the badge doesn't lead to the latest docs as it often happens in other projects.
Having understood that, I checked the correct place and I do indeed see the final change.